zztin/ctimer
# Use message box to fix #12 that was not solve in ctimer_db branch

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
The messages after a clock finishes are now prompt with tikinter default messagebox and simpledialog, solving the problem previously which the main window keep going while not waiting for response from subwindows.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. in concentrationtimer.py: Disable line 30 and enable line 29 temporary for quickly checking results.
2. launch ctimer in terminal (make sure it was installed with developer mode)
3. Start the clock and enters a goal
4. finish the clock
5. Answers the prompt windows by clicking "no"
6. Enter a reason in the next window.
7. Repeat 2-4 and answer yes in the second clock.
8. terminate ctimer app.
9. Check stored database at path: ctimer/data/ctimer.db --> drag and drop in https://inloop.github.io/sqlite-viewer/
10. if the entries you typed are stored correctly, accept the pull request.

## Expected behavior
1. Database is updated as user input.
2. No errors in command line.

## Related Issue
#12 #6
